packages/patternfly-4/react-core/src/experimental/components/OverflowMenu/OverflowMenu.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import styles from '@patternfly/react-styles/css/components/OverflowMenu/overflow-menu';
+import { css, getModifier } from '@patternfly/react-styles';
+
+export interface OverflowMenuProps extends React.HTMLProps<HTMLDivElement> {
+  /** Any elements that can be rendered in the menu */
+  children?: any;
+  /** Additional classes added to the OverflowMenu. */
+  className?: string;
+  /** Indicates breakpoint at which to switch between horizontal menu and vertical dropdown */
+  breakpoint: 'Md' | 'Lg' | 'Xl';
```

Comment:
We use lowercase everywhere else, so `'md' | 'lg' | 'xl'`? Also I believe we have `'sm'` and `2xl`.

Reply:
Thanks - updated to accept lowercase but excluding 'sm' and '2xl' as css is not setup to accommodate these options.